projects/plugins/jetpack/modules/masterbar/admin-menu/class-atomic-admin-menu.php

```diff
@@ -117,9 +117,12 @@ public function get_preferred_view( $screen, $fallback_global_preference = true
 	public function add_plugins_menu() {
 		global $submenu;
 
+		// Calypso plugins screens link.
+		$plugins_slug = 'https://wordpress.com/plugins/' . $this->domain;
+
 		// Hide Add new plugin and plugin file editor menu.
```

Comment:
We could change this comment to explain what's going on here. Something like:
```suggestion
		// `wpcomsh` restricts the plugins capabilities when the site doesn't have a 
		// supported plan so the Plugins menu it's not registered. We still want the 
		// menu to be available though, since it offers an opportunity to buy again
		// the needed plan.
```